cookbooks/swift/recipes/setup.rb

```diff
@@ -71,9 +71,9 @@
   ]
 else
   required_packages += [
-    "python-dev",
```

Comment:
this at least seems required for noble; there is no legacy `python-dev` package anymore - `python3-dev` FTW Huzzah!